akka-projection-jdbc/src/main/scala/akka/projection/jdbc/internal/JdbcOffsetStore.scala

```diff
@@ -70,7 +70,7 @@ private[akka] class JdbcOffsetStore[S <: JdbcSession](
           if (resultSet.first()) {
             val offsetStr = resultSet.getString("OFFSET")
             val manifest = resultSet.getString("MANIFEST")
-            Some(fromStorageRepresentation(offsetStr, manifest))
+            Some(fromStorageRepresentation[Offset](offsetStr, manifest))
```

Comment:
This one was buzzing me for a while. I've seen this warning but never checked the cause. Now I had to fix it.

It was inferred to `Nothing` and compiler was not happy.